core/keeper.go

```diff
@@ -72,24 +72,26 @@ func GetPrioritisedFTSOContract(blockTime *big.Int) string {
 	}
 }
 
-func GetMaximumMintRequest(blockNumber *big.Int) *big.Int {
-	switch {
-	default:
-		maxRequest, _ := new(big.Int).SetString("50000000000000000000000000", 10)
-		return maxRequest
+func GetMaximumMintRequest(blockNumber *big.Int, evm *vm.EVM) *big.Int {
```

Comment:
As we are not using the block number to switch here, but rather the block time, the signature should simply switch to this:

```go
func GetMaximumMintRequest(blockTime *big.Int) *big.Int
```

There is no reason to inject the _entire_ EVM just for a single value.